brave-spancollector-slf4j/src/main/java/com/github/kristofa/brave/slf4j/AsyncSlf4jLoggingSpanCollector.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
```

Comment:
note that at some point we will nuke all of these for "the OpenZipkin or Brave authors" as it is a maintenance burden